### DISCUSSION TITLE
Guilded.gg support

### DISCUSSION THREAD

**Discussion by MenacingPerson** [42wim/matterbridge#1719] [New bridges suggestions]
**Is your feature request related to a problem? Please describe.**
N/A

**Describe the solution you'd like**
Add guilded.gg support

**Describe alternatives you've considered**
N/A

**Additional context**
guilded.gg is a Discord alternative that's turning some heads, but right now the bot api isn't available to the public. I think people would like this.

**Comment by halian** [comment]
I should like to add that Guilded claims to support the Discord webhook API, as touted in the webhook scene of each channel's settings.

**Comment by Ryonez** [comment]
The API is available now.
https://www.guilded.gg/docs/api/introduction

**Comment by halian** [comment]
Bump; the other bridge bots I can find that support Guilded are either feature-poor or discontinued.
